server/src/main/java/org/elasticsearch/cluster/RestoreInProgress.java

```diff
@@ -132,16 +132,12 @@ public Entry(
             List<String> indices,
             Map<ShardId, ShardRestoreStatus> shards
         ) {
+            this.uuid = Objects.requireNonNull(uuid);
             this.snapshot = Objects.requireNonNull(snapshot);
             this.state = Objects.requireNonNull(state);
-            this.quiet = Objects.requireNonNull(quiet);
```

Comment:
Why did we decide to remove this requireNonNull?

Reply:
It's called on a `boolean`, which can never be null